test/pulpcore_plugin_integration_test.rb

```diff
@@ -26,12 +31,13 @@ def test_features
     assert_equal 'running', pulpcore['state'], Proxy::LogBuffer::Buffer.instance.info[:failed_modules][:pulpcore]
 
     expected_settings = {
-      'pulp_url' => 'http://pulpcore.example.com/foo',
-      'content_app_url' => 'http://pulpcore.example.com:24816/',
+      'pulp_url' => 'http://smartproxy.example.com/pulpcore',
+      'content_app_url' => 'http://smartproxy.example.com:24816/',
```

Comment:
Why do you change the hostname? The intention of it was to show that Pulp can technically live on another host and is explicitly not tied to the Smart Proxy hostname.

Reply:
@ekohl that's neat. I'll revise with that intention in mind, thanks